src/schemas/686c-674/schema.js

```diff
@@ -15,6 +15,8 @@ const filteredStates = states.USA.filter(state => !militaryStates.includes(state
 
 let definitions = cloneDeep(commonDefinitions);
 definitions = pick(definitions, 'fullName', 'phone', 'date', 'email', 'files', 'privacyAgreementAccepted', 'ssn');
+definitions.fullName.properties.middle.maxLength = 30;
```

Comment:
I dont know the schema well enough to understand why this line is required. can you explain?